### ISSUE TITLE
Make the version history view clearer

### ISSUE THREAD

**Issue by thebearon** [nextcloud/richdocuments#1782]
**Is your feature request related to a problem? Please describe.**
When you open the version history, and pick different older versions to check in the viewer on the left, there's no clear indication whether you're looking at an older version, and which one it is.
(currently there's a bug in online that the version history is shown in edit mode, which will be fixed shortly)

**Describe the solution you'd like**
Perhaps on the NC sidebar the line of the currently opened version should be highlighted, and/or the document details in the top side of the sidebar should be updated (as a smaller note, the preview doesn't seem to be useful in this case).

In addition/alternatively, online could also indicate in some way that you're looking at the version history, and which time the currently viewed state is from.
@pedropintosilva, what are your thoughts on this from a UX perspective?

Screenshot:
![revision_picker](https://user-images.githubusercontent.com/19438782/134909229-33ee71e7-1a67-41a2-90e4-4e00e1da935b.png)

**Comment by sebastiaanveld** [comment]
Would it be nice if we could leverage the Libreoffice Compare Documents function so we could actually see what is the difference between the two documents. This is something I personally heavily rely on in Wiki's.
Can be found in LO Edit > Track Changes > Compare Document, select an document to compare to the opened document.

**Comment by Jerome-Herbinet** [comment]
> **Is your feature request related to a problem? Please describe.** When you open the version history, and pick different older versions to check in the viewer on the left, there's no clear indication whether you're looking at an older version, and which one it is. (currently there's a bug in online that the version history is shown in edit mode, which will be fixed shortly)
> 
> **Describe the solution you'd like** Perhaps on the NC sidebar the line of the currently opened version should be highlighted, and/or the document details in the top side of the sidebar should be updated (as a smaller note, the preview doesn't seem to be useful in this case).
> 
> In addition/alternatively, online could also indicate in some way that you're looking at the version history, and which time the currently viewed state is from. @pedropintosilva, what are your thoughts on this from a UX perspective?
> 
> Screenshot: ![revision_picker](https://user-images.githubusercontent.com/19438782/134909229-33ee71e7-1a67-41a2-90e4-4e00e1da935b.png)

I totally agree; this lack of marking the selected version is really problematic (especially when the number of versions is large). It would make things much clearer. Just recently, I wanted to restore the old version of a document in Nextcloud Office, and several times before restoring, I didn't know which version of the file I had selected.

**Comment by Jerome-Herbinet** [comment]
I've just made a pull request that would solve this UX issue (small CSS fix).
https://github.com/nextcloud/server/pull/32492
